### PR TITLE
fixes #8204 - deface About page

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: app/controllers/foreman_sam/concerns/settings_controller_extensions.rb [hunk 1]
```diff
@@ -27,7 +27,6 @@ def index
            :email_reply_address,
            :entries_per_page,
            :fix_db_cache,
-           :foreman_url,
```
Comment (thomasmckay) [inline app/controllers/foreman_sam/concerns/settings_controller_extensions.rb]: Setting not applicable to SAM

File: lib/foreman_sam/engine.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@ class Engine < ::Rails::Engine
     end
 
     config.to_prepare do
-      ::SettingsController.send :include, ForemanSAM::Concerns::SettingsControllerExtensions
+      ::SettingsController.send :include, ForemanSAM::Concerns::SettingsControllerExtensions unless Foreman.in_rake?
```
Comment (thomasmckay) [inline lib/foreman_sam/engine.rb]: rake katello:reset (at db:migrate) was failing; this concern not necessary during that rake task

```
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "settings" does not exist
LINE 5:              WHERE a.attrelid = '"settings"'::regclass
                                        ^
:             SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod
              FROM pg_attribute a LEFT JOIN pg_attrdef d
                ON a.attrelid = d.adrelid AND a.attnum = d.adnum
             WHERE a.attrelid = '"settings"'::regclass
               AND a.attnum > 0 AND NOT a.attisdropped
             ORDER BY a.attnum
```

Comment (komidore64) [inline lib/foreman_sam/engine.rb]: why isn't this concern necessary?

Comment (thomasmckay) [inline lib/foreman_sam/engine.rb]: It is necessary during the run, just not during 'rake db:migrate' where it blows up.

Comment (komidore64) [inline lib/foreman_sam/engine.rb]: ok

**Comment by komidore64** [comment]
ack

**Pull request merged**